packages/devextreme/scss/widgets/generic/menu/_index.scss

```diff
@@ -46,6 +46,10 @@
   }
 
   .dx-menu-item-has-text {
+    &.dx-state-focused {
+      color: $menu-color;
```

Comment:
It's also a bit strange to me that we add a separate css rule. I see that we usually use `$menu-item-expanded-color` for expanded menu. Cannot we implement a similar to how it's done in neighbor themes?

Reply:
The reason why I added state-focused rule is because whenever shift+tab happens, the menu focuses and disregards the css rule from menu-item-expanded then uses the css rule for item template. 

If we create a style test specifically for menu-item-expanded, even if we don't have the fix, it would be evergreen but the bug would be still there and still uses item template css.

Adding state-focused rule explicitly targets the focused state and enables us to test it.

Reply:
I mean that it seems that `$menu-item-expanded-color` should be applied in this case, not `$menu-color`. We definitely should add `.dx-state-focused` in some place to increase some selectors specificity and made it applied in our case. But i believe it's not a correct place (moreover, now u can see that u need duplicate this code in 2 places).

I see that this issues is resolved [this way](https://github.com/DevExpress/DevExtreme/blob/24_1/packages/devextreme/scss/widgets/material/menuBase/_index.scss#L49) in Material and Fluent. Cannot we use the same approach?